metadata/utils/data_structures.yml

```
@@ -0,0 +1,17 @@
+  - Misc
+maintainers:
+  - name: Harvey Lynden
+    email: hlynden@redhat.com
+    github_usr_name: harvey0100
+supported_platforms:
+  - CentOS Stream 9
+  - Fedora 36
+  - Fedora 37
+tests:
+  - tests/modules/utils/data_structures.py
+remote: false 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)